keras_cv/visualization/segmentation_mask.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility function to help visualize batch of binary mask.
+"""
+from collections import defaultdict
+
+import tensorflow as tf
+
+from keras_cv.visualization.colors import colors
+
+RGB = 3
+
+
+def _blend(image1, image2, factor=0.4):
+    difference = image2 - image1
+    scaled = factor * difference
+    # Do addition in float.
+    blended = image1 + scaled
+
+    # We need to clip and then cast.
+    blended = tf.round(tf.clip_by_value(blended, 0.0, 255.0))
+    return blended
+
+
+def _map_color_on_mask(masks, color):
+
+    # check distinct mask color codes with color mapping.
+    distinct_mask_code = tf.unique(tf.reshape(masks, -1)).y
+    color_rgb = defaultdict(tuple)
+
+    if isinstance(color, str):
+        for idx in distinct_mask_code:
```

Comment:
Does this work?  Unfortunately this might only work work with eager Tensors.

Reply:
Which might be fine I suppose.

Regardless, you can just use a default dict with the color as a default.  don't need to create a mapping for ever color if you always want the same color as output

Reply:
done